### ISSUE TITLE
Design Intent - Accessibility Feedback - Sitewide Facilities, Facility Locator, Improve Search, Add Filters

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#62285]
## VFS actions
- [ ] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- I think you're working on all the right stuff. No major concerns!
- Definitely continue working with @laflannery! 

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Should:** I don't have any specific objection to using a modal for filters, except that modals are a big enough task interruption that if they can be avoided it would be nice to avoid them. But otherwise I defer to design feedback on consistency with other tools.
- **Should:** Explore other placements for the button that opens the filter modal/dropdown -- or, if possible, validate this placement via usability testing. My main concern is low vision users who use screen magnifiers (as opposed to using browser zoom), who might miss the button as they focus on the left-hand column listing results.
- **Potential accessibility improvements:** To the extent that you are able to prioritize accessibility tech debt, I would love to see:
   - Better focus management when a map marker is clicked. Currently when you click a map marker the results list scrolls to the corresponding entry and adds the `active` class to highlight it. But although the highlight border looks the same as the keyboard focus indicator, it doesn't actually move keyboard focus --- so if I press [tab] I'll be taken somewhere unexpected.
   - Map markers should receive keyboard focus and be keyboard-interactive ([enter] key to activate marker and move focus to the corresponding entry in the results list).
   - Map markers should have a unique accessible name. If they're keyboard accessible, something useful should be announced to screen readers. Map accessibility is a huge pain and it's difficult to create a fully accessible experience for screen reader users. At minimum, screen reader users should know they've tabbed to a map marker that's interactive, and be able to distinguish it from other markers. Better yet would be if there's some way to communicate basic geographic information too --- maybe an address or just a city. Honestly no idea what's possible here, just giving you my wishlist!
   - Another wishlist item: keyboard-accessible controls for moving the map north/south/east/west.
   - If map markers can receive keyboard focus, then provide keyboard users a skip link to jump them past the map and to the pagination (so they don't have to tab through 10 markers plus map controls if they don't want to).

**Comment by mmiddaugh** [comment]
@thejordanwood is this issue still needed or have we addressed all the feedback?

**Comment by thejordanwood** [comment]
@mmiddaugh This feedback has been addressed and I'll keep working with @laflannery on this for any future iterations. Going to close this!
